Build error with 5.0.1

```text
Creating library C:/Libs/ITK-5.0.1-vs2015/lib/Debug/itkIOMeshSTL-5.0.lib and object C:/Libs/ITK-5.0.1-vs2015/lib/Debug/itkIOMeshSTL-5.0.exp
3>itkSTLMeshIO.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: virtual void __cdecl itk::MeshIOBase::SetPointComponentType(enum itk::MeshIOBase::IOComponentType)" (__imp_?SetPointComponentType@MeshIOBase@itk@@UEAAXW4IOComponentType@12@@Z) referenced in function "protected: __cdecl itk::STLMeshIO::STLMeshIO(void)" (??0STLMeshIO@itk@@IEAA@XZ)
```
[Output-Build.txt](https://github.com/InsightSoftwareConsortium/ITKIOMeshSTL/files/3488044/Output-Build.txt)

VS2015x64, CMake 3.15.2, BUILD_SHARED_LIBS ON, and following modules enabled: IOMeshSTL, IOTransformDCMTK, ITKIODCMTK, ITKReview, ITKTBB, ITKVtkGlue and RLEImage.


The link library list is
```text
..\..\..\..\lib\Debug\ITKCommon-5.0.lib
C:\Libs\tbb2019_20190410oss\lib\intel64\vc14\tbb_debug.lib
..\..\..\..\lib\Debug\itksys-5.0.lib
ws2_32.lib
Psapi.lib
..\..\..\..\lib\Debug\ITKVNLInstantiation-5.0.lib
..\..\..\..\lib\Debug\itkvnl_algo-5.0.lib
..\..\..\..\lib\Debug\itkvnl-5.0.lib
..\..\..\..\lib\Debug\itkv3p_netlib-5.0.lib
..\..\..\..\lib\Debug\itknetlib-5.0.lib
..\..\..\..\lib\Debug\itkvcl-5.0.lib
kernel32.lib
user32.lib
gdi32.lib
winspool.lib
shell32.lib
ole32.lib
oleaut32.lib
uuid.lib
comdlg32.lib
advapi32.lib
```
If I manually add `..\..\..\..\lib\Debug\ITKIOMeshBase-5.0.lib` to this list, the module compiles fine.

I cannot spot a problem with [module definition code](https://github.com/InsightSoftwareConsortium/ITKIOMeshSTL/blob/master/itk-module.cmake).

--- a/azure-pipelines.yml
+++ b/azure-pipelines.yml
@@ -1,5 +1,5 @@
 variables:
-  ITKGitTag: v5.0rc02
+  ITKGitTag: v5.0.1
   CMakeBuildType: MinSizeRel
 
 trigger:

--- a/setup.py
+++ b/setup.py
@@ -13,7 +13,7 @@ except ImportError:
 
 setup(
     name='itk-iomeshstl',
-    version='2.0.0',
+    version='2.0.1',
     author='Luis Ibáñez',
     author_email='community@itk.org',
     packages=['itk'],
@@ -48,6 +48,6 @@ setup(
     keywords='ITK InsightToolkit STL Mesh',
     url=r'https://github.com/InsightSoftwareConsortium/ITKIOMeshSTL',
     install_requires=[
-        r'itk>=5.0rc02'
+        r'itk>=5.0.1'
     ]
     )

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -3,7 +3,4 @@ set(IOMeshSTL_SRC
   itkSTLMeshIOFactory.cxx
 )
 
-add_library(IOMeshSTL ${ITK_LIBRARY_BUILD_TYPE} ${IOMeshSTL_SRC})
-target_link_libraries(IOMeshSTL ${ITKCommon_LIBRARIES} ${ITKIOMesh_LIBRARIES})
-
-itk_module_target(IOMeshSTL)
+itk_module_add_library(IOMeshSTL ${ITK_LIBRARY_BUILD_TYPE} ${IOMeshSTL_SRC})

